bin/node-template/runtime/src/lib.rs

```diff
@@ -547,8 +559,8 @@ impl pallet_authorship::Trait for Runtime {
 parameter_types! {
 	pub const SessionsPerEra: SessionIndex = SESSIONS_PER_ERA;
 	pub const BondingDurationInEra: EraIndex = 14 * DAYS
-		/ (SESSIONS_PER_ERA as BlockNumber * BLOCKS_PER_SESSION);
-	pub const BondingDurationInBlockNumber: BlockNumber = 14 * DAYS;
+		/ (SESSIONS_PER_ERA as BlockNumber * BLOCKS_PER_SESSION) + 1;
+	pub const BondingDurationInBlockNumber: BlockNumber = 14 * DAYS + SESSIONS_PER_ERA as BlockNumber * BLOCKS_PER_SESSION;
```

Comment:
The reasons for this change?

Reply:
https://github.com/darwinia-network/darwinia/issues/495